README.md

```diff
@@ -0,0 +1,7 @@
+# koperator-docs
+Documentation for Koperator - the operator for managing Apache Kafka on Kubernetes
+
+This repo contains the documentation for the Koperator, which is the open-source core of SDM.
+
+- The _master_ branch contains the public documentation of Koperator, published at https://banzaicloud.com/docs/supertubes/kafka-operator/
+- The _supertubes-integration_ branch contains the Koperator docs for Calisti. The different branches are needed because of the different release schedule of Koperator and Calisti, and the path/url differences between the public Koperator docs and Calisti. The Calisti docs is published at [https://smm-docs.eticloud.io/sdm/koperator/](https://smm-docs.eticloud.io/sdm/koperator/).
```

Comment:
```suggestion
- The _supertubes-integration_ branch contains the Koperator docs for Calisti. The different branches are needed because of the different release schedules of Koperator and Calisti, and the path/url differences between the public Koperator docs and Calisti. The Calisti documentation is published at [https://smm-docs.eticloud.io/sdm/koperator/](https://smm-docs.eticloud.io/sdm/koperator/).
```